Easydict/objc/StatusItem/EZMenuItemManager.h

```diff
@@ -9,7 +9,7 @@
 #import <Foundation/Foundation.h>
 
 NS_ASSUME_NONNULL_BEGIN
-
+// will deprecation in next release now this class only used to fetch Repo last version in SwiftUI version. In this case it should be removed and move the fetch func to a new class
 @interface EZMenuItemManager : NSObject
```

Comment:
This class is currently only being used by `fetchRepoLatestVersion`. Can you remove the other unused code?

Or you can remove everything and simply rewrite the `fetchRepoLatestVersion` method in Swift.

Reply:
I note this class,  but `EZConfiguration` has a lot of keys, so I think we should remove it in next PR, in next PR will remove `EZConfiguration` and so on. This will make PR is not more complicated.

Reply:
The same, I don't want to make this PR too complicated, so I comment in this place would like to make changes in the next PR.

Reply:
ok

Reply:
@tisfeng How do you think of this idea, this PR I want to try not to be more complicated, only change the deployment version, and try to determine some to modify, in the next PR to improve them.